PowerEditor/src/ScitillaComponent/AutoCompletion.cpp

```diff
@@ -94,19 +95,32 @@ bool AutoCompletion::showApiAndWordComplete()
 	bool canStop = false;
 	for (size_t i = 0, kwlen = _keyWordArray.size(); i < kwlen; ++i)
 	{
-		if (_keyWordArray[i].compare(0, len, beginChars) == 0)
+
+		bool _ignoreCaseMatch = false;
+		if (_ignoreCase)

```

Comment:
This `if` block seems create lots of copies of strings. Is there a better way to compare them? Can you use `_tcsicmp()`? If so could you also use it in `AutoCompletion::sortKeyWords()`?